### DISCUSSION TITLE
GUI layout

### DISCUSSION THREAD

**Discussion by jsuto** [jsuto/piler#61] [Polls]
The GUI has been upgraded to use latest bootstrap and jquery, but the layout is basically the same: 2 horizontal panes that can be resized. However, I got a request to change the layout and perhaps add a dark theme to match the layout with the demo site (https://demo.mailpiler.com) that runs the commercial edition of piler.

Let me know what you think, please vote.

**Comment by thatgeek** [comment]
I have a slight preference for the current layout, but it may just be familiarity. I don't hate the demo site layout, and I could likely get used to it. I think my preference is because we're using mailpiler mostly for searches that touch on multiple recipients mailboxes, and I like having columns for "to" and "from" directly visible in the search results.

**Reply by jsuto** [reply]
Thank you for the insights.

**Comment by fumpitz** [comment]
My problem with the demo site layout is that it requires at least 3 lines for each mail in the list. When I use the archive, I usually have to search for a specific mail in a long list of candidates. In the current layout you get one line ( with long subjects two or more lines) per mail, making better use of the space on the screen.
Just like @thatgeek I also prefer the dedicated columns in the current layout.

It would be great if the width of the columns could be adjusted with the mouse and oversized cell contents are clipped (e.g. using CSS text-overflow: ellipsis ellipsis;) so that each mail is strictly printed in one line.

BR,
Frank

**Reply by jsuto** [reply]
Well, the idea with the demo site's layout was to create something that resembles to a well known mail clent, outlook webapp.
Update: #64 might do it.
